### PR TITLE
Increased Facebook button text limit to 640 from official docs

### PR DESCRIPTION
**Pull request by IvanJov**

### CODE REVIEW

**Comment by stojanovic** [comment]
Limit is 640 now?
Do the same for buttons in generic template please

**Comment by IvanJov** [comment]
Found it here: https://developers.facebook.com/docs/messenger-platform/send-api-reference/button-template

```
text must be UTF-8 and has a 640 character limit
```

**Comment by stojanovic** [comment]
Cool.
It seems that we don't have a limit for [Generic template buttons](https://github.com/claudiajs/claudia-bot-builder/blob/master/lib/facebook/format-message.js#L243-L276), so I can merge this.

File: spec/facebook/facebook-format-message-spec.js [hunk 1]
```diff
@@ -352,7 +352,7 @@ describe('Facebook format message', () => {
     });
 
     it('should throw an error if button text is longer than 320 characters', () => {
```
Comment (stojanovic) [inline spec/facebook/facebook-format-message-spec.js]: Update number here, please :)

**Pull request merged**